driver/cmd/ibm-spectrum-scale-csi/main.go

```diff
@@ -122,7 +132,7 @@ func setContext() context.Context {
 	return ctx
 }
 
-func getLevel() string{
+func getLevel() string {
 	level := os.Getenv(logLevel)
 	var logValue string
 	klog.Infof("logValue: %s", level)
```

Comment:
if env is not set log will show empty value. Better to print after the env check

Reply:
Modified accordingly